pkg/jsonlog/jsonlog.go

```diff
@@ -17,6 +17,8 @@ type JSONLog struct {
 	Created time.Time `json:"time"`
 	// Attrs is the list of extra attributes provided by the user
 	Attrs map[string]string `json:"attrs,omitempty"`
+	// Tags requested the operator
+	Tag string `json:"tag,omitempty"`
```

Comment:
This should be set to `""` in `Reset()`

Reply:
Tag is now initialized to an empty string in Reset().